sema/commons/reason/reason.py

```
@@ -0,0 +1,260 @@
+                    resolved = self._resolve_path(src)
+                    if resolved.is_file():
+                        self._parse_file_into_graph(source_graph, resolved)
+                    elif resolved.is_dir():
+                        for match in sorted(resolved.rglob("*")):
+                            if match.is_file():
+                                try:
+                                    self._parse_file_into_graph(
+                                        source_graph, match
+                                    )
+                                except Exception:
+                                    pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)